news/_posts/2021-01-07-next-official-release-spark-3.1.1.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: "Next official release: Spark 3.1.1" 
+categories:
+- News
+tags: []
+status: publish
+type: post
+published: true
+meta:
+  _edit_last: '4'
+  _wpas_done_all: '1'
+---
+The next official Spark release is Spark 3.1.1 instead of Spark 3.1.0.
+There was an accident during Spark 3.1.0 RC1 preparation,
+see [[VOTE] Release Spark 3.1.0 (RC1)](http://apache-spark-developers-list.1001551.n3.nabble.com/VOTE-Release-Spark-3-1-0-RC1-td30524.html) in the Spark dev mailing list.
```

Comment:
Since this is a reference to Apache project decision, Shall we use the official Apache mailing list link instead of 3rd party URL, `nabble.com`?